src/game/game_main.cpp

```diff
@@ -76,7 +154,7 @@ void update(f32 delta) {
     }
     Util::end_tweak_section(&show_camera_controls);
 
-    static bool show_entity_system = true;
+    static bool show_entity_system = false;
```

Comment:
You can remove this tweak section, it was used to debug the entity system and then just left in here. My bad honestly.